Cython/TestUtils.py

```diff
@@ -396,3 +397,27 @@ def write_newer_file(file_path, newer_than, content, dedent=False, encoding=None
 
     while other_time is None or other_time >= os.path.getmtime(file_path):
         write_file(file_path, content, dedent=dedent, encoding=encoding)
+
+
+def parse_python_code(code):
```

Comment:
This wants merging with py_parse_code that I added with structural pattern matching - the utilities do the same thing so shouldn't be duplicated.